api/app/signals/apps/questionnaires/admin/inlines.py

```diff
@@ -9,7 +9,7 @@
 
 class EdgeStackedInline(NonRelatedStackedInline):
     model = Edge
-    fields = ('question', 'next_question', 'order', 'payload',)
+    fields = ('question', 'next_question', 'order', 'choice',)
```

Comment:
So we always need a choice if we want to take different routes? How de we handle, for example regex based rules in the future? Or will we make that a problem for the next PR :D